posts/verify-sync-release-volunteers/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Call for volunteers: help to test us the release syncing using staging instance"
+date: 2023-08-30T14:46:18+02:00
+authors: lbarczio
+tags:
+  - downstream
+  - pull-from-upstream
+  - propose-downstream
+  - staging
+---
+
+In the upcoming months, we plan to migrate our service to a new cluster. However, this may affect `propose_downstream`
+and `pull_from_upstream` jobs due to the new firewall rules. The problematic aspects could be:
+
+- commands you run in your `actions` during syncing the release involving interactions with external servers
+- downloading your sources from various hosting services (crates.io, npm, gems, etc.)
+
+To smoothen this transition, we kindly encourage you to enable one of these jobs on our already migrated staging instance.
+This recommendation is particularly important if you belong to one of the groups affected by the two previous points.
+This proactive step will help us identify and address any issues promptly. To avoid too much noise in your dist-git PRs,
+you may enable the job for only one target, resulting in only one additional PR created.
+
+Here's how you can enable one of the jobs on the staging instance:
+
+- `pull-from-upstream`:
+  The only thing needed is to duplicate the job in your Packit config using `packit_instances` configuration option. Example:
+
+```yaml
+- job: pull_from_upstream
+  trigger: release
+  packit_instances: ["stg"]
+  dist_git_branches:
+    - fedora-rawhide
+```
+
+- `propose-downstream`:
+  For this job, you first need to [enable our staging Github app](https://github.com/packit/packit/discussions/1530).
```

Comment:
```suggestion
  For this job, you first need to [enable our staging Github app](https://packit.dev/docs/guide#staging-instance).
```

Reply:
I would rather update the part in packit.dev to be up to date so we have all the info there and people going through the onboarding has the full info as well. (It will also be easier to update in the future as well.) And discussions aren't much used I think.